### PR TITLE
Added 2 COBOL compile options

### PR DESCRIPTION
**Pull request by adeanbd**
Created a compiled COBOL module option
Created a compiled COBOL bundle(Module and Program) option

### Changes

Added 2 compile option for COBOL:
{
            "type": "member",
            "extensions": [
                "CBLLE",
                "CBL"
            ],
            "name": "Create ALL-in-one PGM CBLLE (CRTBNDCBL)",
            "command": "CRTBNDCBL (&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OPTION(*SOURCE) DBGVIEW(*SOURCE)"
        },
        {
            "type": "member",
            "extensions": [
                "CBLLE",
                "CBL"
            ],
            "name": "Create Module CBLLE (CRTCBLMOD)",
            "command": "CRTCBLMOD MODULE(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OPTION(*SOURCE) DBGVIEW(*SOURCE)"
        },

### Checklist

* [x] have tested my change
* [ ] updated relevant documentation
* [ ] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in the README
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Comment by worksofliam** [comment]
Hey @adeanbd!

Thanks for a great PR! I'll get this reviewed tomorrow and merged.

File: package.json [hunk 1]
```diff
@@ -308,6 +308,24 @@
 						}
 					},
 					"default": [
+						{
+						    "type": "member",
+						    "extensions": [
+							"CBLLE",
+							"CBL"
+						    ],
+						    "name": "Create ALL-in-one PGM CBLLE (CRTBNDCBL)",
```
Comment (worksofliam) [inline package.json:317]: Can this be renamed to `Create Bound COBOL Program`?

**Review by worksofliam — CHANGES_REQUESTED** [review]
I made one note that I need you to change. Should be good other than that!

File: package.json (hunk 1 above)
Comment (adeanbd) [reply package.json:317]: Hi there,
Yes of course no problem! Sounds more professional ;)

Comment (worksofliam) [reply package.json:317]: @adeanbd Nope! That's it! When you have made the commit to change it I will merge.

**Comment by worksofliam** [comment]
Hey @adeanbd!

Is it possible for you to make the change requested? I'd like to merge this soon!

Thanks

**Review by adeanbd — COMMENTED** [review]
OK to change the text to Create Bound COBOL Program (CRTBNDCBL)

**Comment by adeanbd** [comment]
Hi,
Sorry I'm confuse, and I don't know what to do!
I need to create another Pull request with the changes?
Sorry to slow the process!

**Comment by worksofliam** [comment]
@adeanbd No problem!

I need you to make a commit to your branch/fork to make the change yourself. I cannot make the change on your fork.

**Comment by adeanbd** [comment]
@worksofliam Finally!
Free to go!

**Comment by worksofliam** [comment]
@adeanbd Great work. Thank you!

**Pull request merged**